administrator/components/com_admin/sql/updates/mysql/3.7.0-2016-06-14.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE if NOT EXISTS  `#__share_draft`(
+  `id` int(10) NOT NULL AUTO_INCREMENT COMMENT 'Primary Key to #__content',
+  `title` varchar(255) NOT NULL,
+  `created` datetime NOT NULL,
+  `modified` datetime NOT NULL, 
+  `sharelink` varchar(255) NOT NULL, 
+  PRIMARY KEY  (`id`)
+) DEFAULT CHARSET=utf8;
```

Comment:
@nikitadhiman I am going to pester you once more. The field should be articleId instead of articleid (notice the capital i). The field must also be unsigned because it is impossible this is a negative number. The same unsigned must be set on the primary key field.